data/admin/diagnostic.py

```diff
@@ -45,7 +45,7 @@ class DiagnosticAdmin(SimpleHistoryAdmin):
         "creation_date",
```

Comment:
```suggestion
        "diagnostic_type",
        "creation_date",
```

Pour le voir quand on filtre ?